lib/fastlane/plugin/sentry/actions/sentry_upload_proguard.rb

```diff
@@ -9,17 +9,24 @@ def self.run(params)
         mapping_path = params[:mapping_path]
         android_manifest_path = params[:android_manifest_path]
 
+        has_manifest = !android_manifest_path.nil? && File.file?(android_manifest_path.to_s)
+
         # Verify files
         UI.user_error!("Mapping file does not exist at path: #{mapping_path}") unless File.exist? mapping_path
-        UI.user_error!("AndroidManifest.xml file does not exist at path: #{android_manifest_path}") unless File.exist? android_manifest_path
```

Comment:
AFAIU This validation was never invoked, as the _framework_ checks its valie using ConfigItem's `verify_block` block